src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -155,9 +158,8 @@ typedef struct wrapped_rr_closure_arg {
   /* The RR instance related to the closure */
   grpc_lb_policy *rr_policy;
```

Comment:
We use it in the closure function, on line 171 and following. The closure's arg doesn't have `glb_policy` pointer. Are you suggesting adding a `glb_policy` pointer to the closure arg in order to have access to `rr_policy`?